samples/azure-quantum/hidden-shift/README.md

```diff
@@ -0,0 +1,45 @@
+---
+page_type: sample
+author: cgranade
+description: Find hidden shifts in Boolean functions, using the Azure Quantum service
+ms.author: chgranad@microsoft.com
+ms.date: 01/25/2021
+languages:
+- qsharp
+- python
+products:
+- qdk
+- azure-quantum
+---
+
+# Hidden shift
```

Comment:
Add more detail to help search rank?
Hidden shift of bent functions on Azure Quantum